content/_index.html

```diff
@@ -5,89 +5,176 @@
+          <p class="card-text text-light">
+            Kubeflow has a built-in <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> job operator that you can use to
+            train a machine learning model with TensorFlow. The training
+            controller can be configured to use various compute resources (CPUs
+            or GPUs) and is compatible with various cluster sizes.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/Models.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Model Serving image"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Model Serving</h5>
+          <p class="card-text text-light">
+            Kubeflow supports a <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> Serving container to export trained
+            TensorFlow models to Kubernetes. Kubeflow is also integrated with
+            <a target='_blank' class='text-light' href="https://www.seldon.io/">Seldon Core <a/>, an open source platform for deploying machine learning
+            models on Kubernetes, and <a target='_blank' class='text-light' href="https://www.developer.nvidia.com/tensorrt/">NVIDIA TensorRT</a> Inference Server for
+            maximized GPU utilization when deploying ML/DL models at scale.
+          </p>
+        </div>
+      </div>
+    </div>
+    <br />
+    <div class="card-deck">
+      <div class="card border-primary">
+        <img
+          src="logos/Pipeline.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Pipelines Logo"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Pipelines</h5>
+          <p class="card-text text-light">
+			<a target='_blank' class='text-light' href="https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/" >Kubeflow’s pipeline dashboard</a> is a comprehensive solution to deploy
+            and manage end-to-end machine learning workflows, providing rapid
+            and reliable ML experimentation. Each run is heavily detailed and
+            the ability to schedule runs is also included.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/MultiPlatform.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Multiple Framework Logos"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Multi-framework</h5>
+          <p class="card-text text-light">
+            Our development plans exist beyond <a target='_blank' class='text-light' href="https://www.tensorflow.org/"=>TensorFlow </a>, and we are working
```

Comment:
<I'll come back to reviewing this card when we've decided what to do about the training and serving components mentioned in my earlier comments.>

Reply:
Based on abhi-g's comments, we can keep the allocation of components to cards as it is now, and confine our updates to the wording. Nice!

Suggestions for this card:

* s/exist/extend
* Remove the extra space between "TensorFlow" and the comma. (Please check weird spacing in the links over the entire page - I know you mentioned this in our in-person discussions on Friday.)
* Actually, change the comma to a full stop (period) and start a new sentence.
* Change this: "we are working hard in continuing to include PyTorch..." to this: "We're working hard to extend the support of PyTorch..."

Reply:
Please would you check and fix the brand names on the page: check that the capitalization matches what the brand owner uses in their docs (e.g. Istio) and that we have the correct brand name (e.g. Seldon or Seldon Core).